storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -1,51 +1,51 @@
 [NOTE]
 ====
-If you installed the AWS EBS CSI Operator and driver on an {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
+If you installed the Amazon EBS CSI Operator and driver on an {product-title} 4.5 cluster, you must uninstall the 4.5 Operator and driver before you update to {product-title} {product-version}.
 ====
 endif::openshift-rosa,openshift-dedicated[]
 
 include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-{product-title} defaults to using the CSI plugin to provision AWS EBS storage.
+{product-title} defaults to using the CSI plugin to provision Amazon EBS storage.
 ====
 
-For information about dynamically provisioning AWS EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using AWS Elastic Block Store].
+For information about dynamically provisioning Amazon EBS persistent volumes in {product-title}, see xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using Amazon Elastic Block Store].
 
 ifndef::openshift-rosa,openshift-dedicated[]
 include::modules/persistent-storage-byok.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-If there is no encrypted key defined in the storage class, only set `encrypted: "true"` in the storage class. The AWS EBS CSI driver uses the AWS managed alias/aws/ebs, which is created by Amazon EBS automatically in each region by default to encrypt provisioned storage volumes. In addition, the managed storage classes all have the `encrypted: "true"` setting.
+If there is no encrypted key defined in the storage class, only set `encrypted: "true"` in the storage class. The Amazon EBS CSI driver uses the AWS managed alias/aws/ebs, which is created by Amazon EBS automatically in each region by default to encrypt provisioned storage volumes. In addition, the managed storage classes all have the `encrypted: "true"` setting.
 ====
 
-For information about installing with user-managed encryption for AWS EBS, see xref:../../installing/installing_aws/installing-aws-customizations.adoc#installation-configuration-parameters_installing-aws-customizations[Installation configuration parameters].
+For information about installing with user-managed encryption for Amazon EBS, see xref:../../installing/installing_aws/installing-aws-customizations.adoc#installation-configuration-parameters_installing-aws-customizations[Installation configuration parameters].
 endif::openshift-rosa,openshift-dedicated[]
 
 [role="_additional-resources"]
-== Additional resources
-* xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using AWS Elastic Block Store]
+.Additional resources
+* xref:../../storage/persistent_storage/persistent-storage-aws.adoc#persistent-storage-aws[Persistent storage using Amazon Elastic Block Store]
```

Comment:
I have updated on Line 50. Please let me know if this instance need to be revoked. Thank you!

Reply:
Revoked. [Comment](https://github.com/openshift/openshift-docs/pull/59650#pullrequestreview-1436149688)